index.js

```diff
@@ -1,13 +1,99 @@
+const $hamburger = document.querySelector('#hamburger');
+const $navbar = document.querySelector('.nav-links');
+const $closeMenuButton = document.querySelector('.nav-links img');
+const $linksMenu = document.querySelectorAll('.nav-links li a');
+const $modal = document.querySelector('.modal');
+const $modalTitle = document.querySelector('#modal-title');
+const $pModal = document.querySelector('#p-modal');
+const $imgModal = document.querySelector('#img-modal');
+const $liModal = document.querySelectorAll('.li-modal');
+const $openModal = document.querySelectorAll('.button-project');
+const $closeModal = document.querySelector('.close-button');
+const frstImagePath = '/resources/work-section/masks/Snapshoot PortfolioTonic-mask.png';
+const scndImagePath = '/resources/work-section/masks/Snapshoot PortfolioMulti-Post-mask.png';
+const thrdImagePath = '/resources/work-section/masks/Snapshoot PortfolioTonic2-mask.png';
+const frthImagePath = '/resources/work-section/masks/Snapshoot PortfolioMulti-Post2-mask.png';
+let scrollEnabled = true;
+const projectsInfo = [{
+  name: 'Tonic',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: frstImagePath,
+}, {
+  name: 'Multi Post Stories',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: scndImagePath,
+}, {
+  name: 'Tonic',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: thrdImagePath,
+}, {
+  name: 'Multi-Post Stories',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: frthImagePath,
```

Comment:
- [ ] Kindly ensure that the image spans the width of the container correctly.

- Your design
<img width="797" alt="image" src="https://github.com/SaveryIV/SaveryIV.github.io/assets/41163571/1aa145a1-51c3-475b-b7ec-e4f265fd05ae">

[Check this out for more information](https://stackoverflow.com/questions/3029422/how-to-auto-resize-an-image-while-maintaining-aspect-ratio), let the image be in container give the image a 100% width and height then resize the container proportionally. 


- Figma design
<img width="327" alt="image" src="https://github.com/SaveryIV/SaveryIV.github.io/assets/41163571/74519cf4-b0ef-4cba-8227-48f6c43484dc">